src/systems/ec/implementations/react/packages/link/Link.jsx

```diff
@@ -14,19 +14,28 @@ const Link = ({
   className,
   ...props
 }) => {
+  const iconMarkup =
+    icon.constructor === Array ? (
```

Comment:
`Array.isArray(icon)` is better 
— https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

Reply:
it seems to be slower... I followed the comment here: https://stackoverflow.com/a/26633883